terraform-modules/k8s-master/variables.tf

```diff
@@ -1,25 +1,31 @@
 # See: https://github.com/hashicorp/terraform/issues/21418#issuecomment-495818852
 variable dependencies {
-  type = any
-  default = []
+  type        = any
+  default     = []
   description = "Work-around for Terraform 0.12's lack of support for 'depends_on' in custom modules."
 }
 
 variable name {
-  type = string
+  type        = string
   description = "Name to assign to the master / GKE cluster."
 }
 
 variable location {
-  type = string
+  type        = string
   description = "Zone or region to host the cluster. NOTE: passing a region here will give you a regional cluster with 3x the number of nodes."
 }
 
 variable version_prefix {
-  type = string
+  type        = string
   description = "Version-prefix of k8s to use in the cluster (i.e. '1.14.')."
 }
 
+variable release_channel {
+  type        = string
+  description = "Release channel - RAPID, REGULAR, or STABLE"
+  default     = "UNSPECIFIED"
```

Comment:
Is there a specific option we should be enforcing rather than unspecified, or are you trying to enforce that this value is provided in tfvars?

Reply:
Probably STABLE would be a good default, but I wanted to avoid enabling release channels on random K8s clusters for other people who are using this module. (My understanding based on the docs is that UNSPECIFIED yields the same result as not passing a release_channel block, which is what the module was doing before)